lib/auth.js

```diff
@@ -478,52 +304,6 @@ api.resolveIdentitySlug = function(name, options, callback) {
 };
 
 /**
- * Gets the Identity IDs that match the given identifier. The identifier
- * can be an Identity ID, an Identity slug, or an email address.
- *
- * @param identifier the identifier to resolve.
- * @param callback(err, identityIds) called once the operation completes.
- */
-api.resolveIdentityIdentifier = function(identifier, callback) {
```

Comment:
This is now in `bedrock-authn-password` -- let's mark it deprecated and call into that module and make this a simple alias.